concepts/05 Widgets/CheckBox/10 Handle the Value Change Event.md

```diff
@@ -18,16 +18,16 @@ To process a new **CheckBox** value, you need to handle the value change event.
 
     <!--HTML-->
     <dx-check-box
-        (onValueChanged)="checkBox_valueChanged($event)">
+        [(onValueChanged)]="handleValueChange($event)">
     </dx-check-box>
 
     <!--TypeScript-->
     import { DxCheckBoxModule } from "devextreme-angular";
     // ...
     export class AppComponent {
-        checkBox_valueChanged (e) {
-            let previousValue = e.previousValue;
-            let newValue = e.value;
+        handleValueChange (e) {
```

Comment:
Same notes here as in the [DateBox PR](https://github.com/DevExpress/devextreme-documentation/pull/362#pullrequestreview-362500110).